paperqa/settings.py

```diff
@@ -405,7 +405,7 @@ class IndexSettings(BaseModel):
         description="Whether to recurse into subdirectories when indexing sources.",
     )
     concurrency: int = Field(
-        default=5,  # low default for folks without S2/Crossref keys
+        default=2,  # low default for folks without S2/Crossref keys
```

Comment:
Why are we changing the default in this PR?

Reply:
When I ran the test with concurrency set to 4 I got 429 from semantic scholar.

Reply:
Then let's make the particular test configure a lower value, without changing the default